lib/openstax_path_prefixer.rb

```diff
@@ -14,6 +14,12 @@ def self.configuration
 
     class Configuration
       attr_accessor :prefix
+      attr_accessor :prefix_assets
+
+      def initialize
+        @prefix = ""
+        @prefix_assets = Rails.env.test? ? true : false
+      end
```

Comment:
this is enabling it in `development` mode right?  But above in the README, you're suggesting 
`config.prefix_assets = Rails.env.production?` 

Should the two behaviors match?

Reply:
This is just enabling it in `test` mode so that I can test that it works.